winui/Ribbon/Getting-Started.md

```diff
@@ -95,7 +95,7 @@ this.rootGrid.Children.Add(sfRibbon);
 {% endhighlight %}
 {% endtabs %}
 
-![Add ribbon tab in ribbon](Getting-Started-images/add-ribbon-tab.png)
+![WinUI Ribbon tab added in ribbon](getting-started-images/winui-ribbon-added-tab.png)
```

Comment:
Change alt text like as 
Add Ribbon Tab in WinUI application.
Change image name as per alt text.

Reply:
Changed